oxidation/bindings/generator/templates/binding_web_meths.rs.j2

```diff
@@ -58,7 +58,7 @@ let {{ param_name }} =
 {
     let custom_from_rs_string = {{ type.custom_from_rs_string }};
     custom_from_rs_string({{ param_name }})
-        .map_err(|e| TypeError::new(&e.to_string()))
+        .map_err(|e| TypeError::new(AsRef::<str>::as_ref(&e)))
```

Comment:
Why don't you call `.as_ref()`, simply ? `TypeError` needs `&str` so you are sure that `as_ref` must be from `AsRef<str>`

Reply:
done